servicecontrol/plugins/index.md

```diff
@@ -13,15 +13,32 @@ NOTE: When ServiceControl is introduced into an existing environment the standar
 
 Plugins collect information from NServiceBus and are deployed together with each NServiceBus endpoint. These plugins are optional from the perspective of the NServiceBus framework itself (they are not required by the endpoint), but they are required in order to collect the information that enables ServiceControl (and its clients) to provide the relevant functionality for each plugin.
 
+## Functionalities
 
-## Installing and Deploying
+ServiceControl provide monitoring by analyzing configured audit and fault queues. Based on audit and fault messages it can extract information like: endpoint names, queue names, exception stack traces etc. Based on that information ServiceControl stores them in built-in database and exposes them to 3rd party tools.
```

Comment:
@WojcikMike: Instead of adding a new heading called Functionalities, how about moving this to end of the 1st paragraph in this article? 

Also about the following instead:
ServiceControl provides the monitoring capability by analyzing the configured error and the audit queues. It can extract information like endpoint name, queue name and in case of error messages the exception stack trace, etc.  This information is stored in a built-in internal database.